interfaces/builtin/opengl.go

```diff
@@ -24,8 +24,8 @@ import (
 )
 
 const openglConnectedPlugAppArmor = `
-# Description: Can access opengl. 
-# Usage: reserved
+# Description: Can access opengl.
```

Comment:
This makes me think it would be really cool if we would auto-generate the interfaces.md from the comments in the apparmor profile. Too much work for too little gain at this point probably.